book/src/proposals/one-round-finality.md

```diff
@@ -0,0 +1,104 @@
+# Practical Finality in One Round of Consensus
+
+The cluster can achieve a high degree of finality though Tower
+consensus, and the cluster commits to an exponentially increasing
+of safety for a block.  This safety is based on the commitment of
+any validator from switching votes to an alternative fork.  It is
+possible to achieve finality a different way, leaders can agree not
+to propose alternative forks.
+
+If only a part of the cluster has agreed not to propose alternative
```

Comment:
Are you saying that by knowing the % of a cluster that hasn't 'agreed' to not propose alternative forks, we are able to calculate the likelihood of a votable alternative fork?

Reply:
@ericlwilliams more than just the percentage.  The set of all the leaders that haven't voted and the leader schedule can be used to simulate how lockout will grow in the future.  The assumption I am making is that the partition is somewhat static, and the same leaders will continue voting.